.gitignore

```diff
@@ -5,3 +5,4 @@
 *.user
 /RetroAchievementsKey
 test_app/
+packages/
```

Comment:
This needs to be reverted, ignore your packages locally if you are testing things.